graphql/schema.graphql

```diff
@@ -1,8 +1,20 @@
+input MessageTranslation {
+  id: String!
+  content: String
+  description: String
+}
+
+input MessagesFromProvider {
+  provider: String!
+  messages: [MessageTranslation]
+}
+
+# Apparently you can only add a mutation if you have a query
 type Query {
-  # Apparently you can only add a mutation if you have a query
-  getProductTranslationsCSV: String
+  MOCK_DO_NOT_CALL: String
 }
 
+
 type Mutation {
-  addProductTranslationsCSV(csv: Upload!): Boolean
+  addProductTranslations(translations: [MessagesFromProvider]!, translateTo: String!): [String]!
```

Comment:
[⛏] I think `language` or `targetLanguage` would be clearer than `translateTo`, what do you think?

Reply:
Also, maybe creating a type to make the return of this mutation more explicit would improve readability. `String` is just too generic. Something like
```
type FailedTranslation {
  id: String
}
```

Reply:
Good ones :)